bin/nic.pl

```diff
@@ -101,12 +101,12 @@
 exitWithError("I can't live without a project name! Aieeee!") if !$project_name;
 $clean_project_name = cleanProjectName($project_name);
 
-$package_name = $package_prefix.".".lc($clean_project_name) if $CONFIG{'skip_package_name'};
-promptIfMissing(\$package_name, $package_prefix.".".lc($clean_project_name), "Package Name") unless $NIC->variableIgnored("PACKAGENAME");
+$package_name = $package_prefix.".".$clean_project_name if $CONFIG{'skip_package_name'};
+promptIfMissing(\$package_name, $package_prefix.".".$clean_project_name, "Package Name") unless $NIC->variableIgnored("PACKAGENAME");
```

Comment:
These changes introduce a dpkg control file incompatibility and do not affect the directory name.